### PR TITLE
test/cql-pytest: print driver version

### PR DESCRIPTION
**Pull request by alecco**
Print driver version for cql-pytest tests.

Signed-off-by: Alejo Sanchez <alejo.sanchez@scylladb.com>

### CODE REVIEW

File: test/cql-pytest/conftest.py
Comment (denesb) [inline test/cql-pytest/conftest.py:26]: why warning?

**Comment by alecco** [comment]
Changes in https://github.com/scylladb/scylladb/commit/304564bc0cafa6ed834a13ffa51fc07fd73edff6:

- Fix log level (bdenes)
- Improve format

**Review by denesb — APPROVED** [review]
LGTM, but as this is @nyh's baby, I'd like his approval before merging.

**Review by nyh — APPROVED** [review]
I think logger is an overkill here - a simple single-line "print" would have been simpler and look better in the output .
But I can understand where you're coming from, wanting to use "standard" Python stuff like the logging package, and probably also use it later in other tests. Oh, well ;-) I'll approve.
In any case, I agree that the information conveyed by this message is important, and I often wondered myself which driver I was really using (because various Scylla installation scripts tend to mess which whichever driver I thought I was using).

File: test/cql-pytest/conftest.py
Comment (gusev-p) [inline test/cql-pytest/conftest.py:25]: Do we really need `basicConfig` here? I though that `pytest.ini` is the main tool to configure logs in `pytest`.

Comment (nyh) [reply test/cql-pytest/conftest.py:25]: I personally prefer this approach (the code in conftest.py). What's the point of moving a small part of the logic into a separate file pytest.ini? It's not like we can move anything else there, so what's the benefit of moving this specific command? 
Note that currently test/cql-pytest/pytest.ini is currently empty.

Comment (nyh) [reply test/cql-pytest/conftest.py:25]: By the way, I'm really curious what sort of benefit do you guys see in using "logging" in pytest over just using "print" everywhere. pytest already hides all the output unless the test fails, which is what you almost always want to do anyway - the concept of "log levels" seems to me a mismatch to testing.

**Comment by alecco** [comment]
Changes in https://github.com/scylladb/scylladb/commit/f5b4716a80f009f0baa126b0549609874a098254:

- Use `print()` instead of logger

File: test/cql-pytest/conftest.py
Comment (nyh) [inline test/cql-pytest/conftest.py:24]: Silly nitpick - why not a single line? Something like
print(f"Driver used: {DRIVER_NAME} {DRIVER_VERSION}")

**Review by nyh — APPROVED** [review]

**Review by gusev-p — APPROVED** [review]

**Review by kostja — APPROVED** [review]

**Comment by alecco** [comment]
Review ping   @nyh

**Comment by nyh** [comment]
> Review ping @nyh

I actually approved it 3 weeks ago but then forgot to merge it (I guess I was waiting for the CI?). I'll do it now.

**Pull request closed without merging**
